data/json/flags.json

```diff
@@ -1689,6 +1689,10 @@
     "id": "SPEEDLOADER",
     "type": "json_flag"
   },
+  {
+    "id": "SPEEDLOADER_CLIP",
+    "type": "json_flag"
```

Comment:
Might be worth adding `"info"` entry here to inform player e.g.; "speedloader_clip can be used to quickly reload magazines yadda yadda..." (without getting into clip vs magazine arguments 😂 ), although we're now in string freeze so that might be too annoying as that'd postpone the merge

Or update the docs https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/JSON_FLAGS.md#magazines (or both)